packages/beacon-node/src/util/dataColumns.ts

```diff
@@ -128,7 +128,10 @@ export function getValidatorsCustodyRequirement(
   }, 0);
 
   const count = Math.floor(totalNodeEffectiveBalance / config.BALANCE_PER_ADDITIONAL_CUSTODY_GROUP);
-  return Math.min(Math.max(count, config.VALIDATOR_CUSTODY_REQUIREMENT), NUMBER_OF_CUSTODY_GROUPS);
+  return Math.max(
+    Math.min(Math.max(count, config.VALIDATOR_CUSTODY_REQUIREMENT), NUMBER_OF_CUSTODY_GROUPS),
+    config.NODE_CUSTODY_REQUIREMENT
+  );
```

Comment:
I see what you are doing here.  This is a tricky bit but its also confusing for future maintainers.  Might be easier to read through if we break it over a few lines with some variable names that describe the steps.